dist/style.css

```diff
@@ -0,0 +1,297 @@
+* {
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: linear-gradient(
+    110.1deg,
+    rgba(34, 126, 34, 1) 29%,
+    rgba(168, 251, 60, 1) 90.3%
+  );
+}
+
+header {
+  width: 100vw;
+  height: 120px;
+  text-align: center;
+  padding-top: 1rem;
+  text-shadow: 2px 2px 8px #000;
+}
+
+header h1 {
+  font-size: 3rem;
+  color: #fff;
+}
+header p {
+  color: #fff;
+}
+
+.header-selectors {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80px;
+}
+
+.header-selectors > select,
+.header-selectors > input[type="text"] {
+  padding: 0.5rem;
+  border: none;
+  font-size: 1rem;
+  margin-right: 1.5rem;
+  margin-left: 0.5rem;
+  color: #000;
+}
+.header-selectors > input[type="text"] {
+  margin-right: 0;
+}
+
+.option-list {
+  color: #000;
+}
+
+main {
+  display: flex;
+  width: 100vw;
+}
+
+footer {
+  width: 100vw;
+  color: #000;
+  display: flex;
+  justify-content: center;
+}
+
+footer a {
+  margin: 0 5px;
+  color: #000;
+}
+
+.cards-container {
+  width: 100%;
+  display: flex;
+  margin: 2rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: left;
+}
+
+.project-card {
+  color: #000;
+  flex-basis: 475px;
+  background-color: rgba(255, 255, 255, 0.98);
+  border-radius: 15px;
+  padding: 25px;
+  border: 1px solid rgb(6, 46, 14);
+  position: relative;
+  padding-bottom: 60px;
+  backdrop-filter: blur(50px);
+  margin-bottom: 15px;
+  margin-left: 15px;
+  flex-grow: 3;
+}
+
+.project-card h2 {
+  color: #000;
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.project-card li {
+  list-style: none;
+  color: #000;
+  padding: 5px 0;
+  border-bottom: 1px solid red;
+}
+.project-card li:last-child {
+  border-bottom: none;
+}
+
+.project-card li input[type="checkbox"] {
+  height: 20px;
+  width: 20px;
+  margin-right: 10px;
+  vertical-align: middle;
+  cursor: pointer;
+}
+
+.project-card li label {
+  cursor: pointer;
+}
+
+.project-card li input[type="checkbox"]:checked {
+  height: 10px;
+  width: 10px;
+  position: relative;
+  left: 5px;
+}
+
+.project-card li input[type="checkbox"]:checked + label {
+  margin-right: 5px;
+  vertical-align: middle;
+  text-decoration: line-through;
+  opacity: 50%;
+  font-size: 0.75rem;
+}
+
+.project-card button {
+  position: absolute;
+  height: 50px;
+  bottom: -0.5;
+  left: 0;
+  border-radius: 0 0 15px 15px;
+  width: 100%;
+  border: 0;
+  color: #fff;
+  cursor: pointer;
+  font-size: 1rem;
+  outline-style: none;
+  background-color: green;
+}
+
+.project-card button:hover {
+  background-color: greenyellow;
+  color: #000;
+}
+
+/* .project-card button:focus {
+  background-color: greenyellow;
+  color: #000;
+} */
+
```

Comment:
## Remove Commented out code
- [ ] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))